### DISCUSSION TITLE
안녕하세요. 질문드립니다.

### DISCUSSION THREAD

**Discussion by likespike** [yoonjeong-kwon/fastcampus-kubernetes#17] [Q&A]
안녕하세요. 왕초보라 막혀 질문드립니다.

Ch.2-2 Pod-컨테이너로 환경변수 전달 실습을 따라하고 있습니다. 다음과 같은 error가 발생합니다. 어떻게 해야 하나요?

error: error validating "hello-app.yaml": error validating data: ValidationError(Pod.spec.containers[0]): unknown field "images" in io.k8s.api.core.v1.Container; if you choose to ignore these errors, turn validation off with --validate=false

감사합니다.

**Comment by likespike** [comment]
아 해결했습니다. images라고 잘못 입력했네요;;

**Reply by yoonjeong-kwon** [reply]
@likespike 님, 안녕하세요. 잘 해결되셨다니 다행입니다! 😃
